nfcd/src/main/java/de/tu_darmstadt/seemoo/nfcgate/xposed/InjectionBroadcastWrapper.java

```diff
@@ -27,7 +28,11 @@ public InjectionBroadcastWrapper(Context ctx) {
         // start broadcast receiver on handler thread
         HandlerThread ht = new HandlerThread("ht");
         ht.start();
-        ctx.registerReceiver(this, new IntentFilter("de.tu_darmstadt.seemoo.nfcgate.daemoncall"), null, new Handler(ht.getLooper()));
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            ctx.registerReceiver(this, new IntentFilter("de.tu_darmstadt.seemoo.nfcgate.daemoncall"), null, new Handler(ht.getLooper()), Context.RECEIVER_EXPORTED);
```

Comment:
`Context.RECEIVER_EXPORTED` is only defined on SDK level >= 33 (T). The `SecurityException` is only thrown for apps targeting SDK level >= 34 (U), so this should be fine.
```suggestion
        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
            ctx.registerReceiver(this, new IntentFilter("de.tu_darmstadt.seemoo.nfcgate.daemoncall"), null, new Handler(ht.getLooper()), Context.RECEIVER_EXPORTED);
```